code/game/objects/items/weapons/material/swiss.dm

```diff
@@ -113,12 +113,18 @@
 		. = ..()
 		update_force()
 		return
-	if(istype(target, /obj/item))
-		if(target.w_class <= ITEM_SIZE_NORMAL)
-			can_use_tools = TRUE
-			. = ..()
-			can_use_tools = FALSE
+	if(istype(target, /obj))
```

Comment:
Check for target size

Reply:
for what reason? If it checks for target size you would be unable to use the screwdriver for example on things like a door or window, wouldn't you?